.github/workflows/ci_zkevm.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - zkevm
       - zkevm-2.60
+      - stable*
```

Comment:
I believe, the regex for this should be stable/**

Reply:
thanx! done